src/Common/src/System/Net/SecurityProtocol.cs

```diff
@@ -8,7 +8,11 @@ namespace System.Net
 {
     internal static class SecurityProtocol
     {
-        public const SslProtocols DefaultSecurityProtocols = SslProtocols.Tls | SslProtocols.Tls11 | SslProtocols.Tls12;
+        public const SslProtocols DefaultSecurityProtocols =
+#if !netstandard && !netfx
+            ((SslProtocols)12288) |
```

Comment:
where did the magic number '12288' come from? can't we use a named constant instead to help document what this is?